db/migrate/20180410071554_current_policy.rb

```diff
@@ -0,0 +1,58 @@
+Sequel.migration do
+  up do
+    alter_table :policy_versions do
+      set_column_type :created_at, :timestamptz
```

Comment:
For my edification: What is `timestamptz` and what makes it a better choice?

Reply:
`timestamptz` is short for `TIMESTAMP WITH TIME ZONE` which is exactly the same as `TIMESTAMP WITHOUT TIME ZONE` except if you for some reason use local time instead of UTC, Postgres translates it accordingly on I/O and always stores UTC in the database.

We ought to use UTC everywhere anyway, but this assumption has proven to be a source of a couple bugs and mistifying test failures.